docs/documentation.rst

```diff
@@ -298,52 +298,8 @@ If you spot segmentation issues, manually fix them using the procedure described
 below. Also see the video tutorial after the procedure.
 
 - Go to the ``results/data`` folder
-- Create the file ``fix_seg.sh`` and copy/past the code below:
-
-.. code-block:: bash
-
-  #!/bin/bash
-  # Folder to output the manual labels
-  PATH_SEGMANUAL="../../seg_manual"
-  mkdir $PATH_SEGMANUAL
-  # List of files to correct segmentation on
-  FILES=(
-  sub-amu02_acq-T1w_MTS.nii.gz
-  sub-beijingGE04_T2w_RPI_r.nii.gz
-  sub-brnoPrisma01_T2star_rms.nii.gz
-  sub-geneva04_dwi_crop_moco_dwi_mean.nii.gz
-  )
-  # Loop across files
-  for file in ${FILES[@]}; do
-    # extract subject using first delimiter '_'
-    subject=${file%%_*}
-    # check if file is under dwi/ or anat/ folder and get fname_data
-    if [[ $file == *"dwi"* ]]; then
-      fname_data=$subject/dwi/$file
-    else
-      fname_data=$subject/anat/$file
-    fi
-    # get fname_seg depending if it is cord or GM seg
-    if [[ $file == *"T2star"* ]]; then
-      fname_seg=${fname_data%%".nii.gz"*}_gmseg.nii.gz${fname_data##*".nii.gz"}
-      fname_seg_dest=${PATH_SEGMANUAL}/${file%%".nii.gz"*}_gmseg-manual.nii.gz${file##*".nii.gz"}
-    else
-      fname_seg=${fname_data%%".nii.gz"*}_seg.nii.gz${fname_data##*".nii.gz"}
-      fname_seg_dest=${PATH_SEGMANUAL}/${file%%".nii.gz"*}_seg-manual.nii.gz${file##*".nii.gz"}
-    fi
-    # Copy file to PATH_SEGMANUAL
-    cp $fname_seg $fname_seg_dest
-    # Launch FSLeyes
-    echo "In FSLeyes, click on 'Edit mode', correct the segmentation, then save it with the same name (overwrite)."
-    fsleyes -yh $fname_data $fname_seg_dest -cm red
-  done
-
-- Make this script executable:
-
-  .. code-block:: bash
-
-     chmod 775 fix_seg.sh
-
+- Edit list of files to correct segmentation on, i.e. edit ``FILES`` variable in ``manual_correction.sh`` script
```

Comment:
@valosekj if we add this script in the git repository, it would be more elegant to _not_ have the user edit this file (otherwise it will create conflicts with the origin/master branch). Instead, how about you slightly modify the script to accept `FILES` as an input argument. So the procedure would be:
- in the terminal, define the list of files to correct, example:
~~~
FILES=(
sub-amu02_acq-T1w_MTS.nii.gz
sub-beijingGE04_T2w_RPI_r.nii.gz
sub-brnoPrisma01_T2star_rms.nii.gz
sub-geneva04_dwi_crop_moco_dwi_mean.nii.gz
)
~~~
- Then, call the script with the list of files as input argument:
~~~
PATH_TO_SPINEGENERIC/processing/manual_correction.sh $FILES
~~~

Reply:
Absolutely! I was thinking about passing of this variable as input argument as well.

Reply:
Implemented :+1: , if you agree with it, I will update doc.

Reply:
@valosekj i thought we are using a yml file now? the doc should be updated